config/application.rb

```diff
@@ -10,6 +10,14 @@ module BookReview
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
+    config.time_zone = 'Moscow'
+
+    # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
+    config.i18n.load_path += Dir[Rails.root.join('ru', 'locales', '*.{rb,yml}').to_s]
```

Comment:
```
Dir[Rails.root.join("config", "locales", "**", "*.{rb,yml}")]
```
This command will load files recursively, so you can use subfolders for locale files